packages/react-devtools-shared/src/constants.js

```diff
@@ -11,7 +11,7 @@
 export const __DEBUG__ = false;
 
 // Flip this flag to true to enable performance.mark() and performance.measure() timings.
-export const __PERFORMANCE_PROFILE__ = false;
+export const __PERFORMANCE_PROFILE__ = true;
```

Comment:
Did we intend to flip this flag to true in this PR?

Reply:
Whoops! No. Good catch.